lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -15,26 +15,31 @@ def self.determine_next_version_using_labels(repo_name, github_token, rate_limit
         commits = Helper::GitHubHelper.get_commits_since_old_version(github_token, old_version, repo_name)
 
         type_of_bump = :patch
-
+        has_public_changes = false
+        public_change_labels = %w[breaking docs feat fix perf]
```

Comment:
PCH's PRs that update iOS/Android will have `build` (https://github.com/RevenueCat/purchases-hybrid-common/pull/208) as the `label`. In that case we would also want to make a new release.

In general maybe not all `build` PRs though, so I wonder if we should change that to `dependencies`, and include that in this list. What do you think?

Reply:
Ah great catch! Yeah, your idea to separate them sounds good to me

Reply:
Great catch. Will update those to be dependencies instead 😄